x/btcstaking/types/incentive.go

```diff
@@ -36,28 +36,44 @@ func (dc *VotingPowerDistCache) FindNewActiveFinalityProviders(prevDc *VotingPow
 }
 
 // ApplyActiveFinalityProviders sorts all finality providers, counts the total voting
-// power of top N finality providers, and records them in cache
+// power of top N finality providers, excluding those who don't have timestamped pub rand
+// and records them in cache
 func (dc *VotingPowerDistCache) ApplyActiveFinalityProviders(maxActiveFPs uint32) {
```

Comment:
minor: can we have some separate table driven test case for this function with few possbile scenarios like:
- few finality provider timestamped, few don't
- different number of active inactive 
...

A lot is riding on this function being correct so it would be nice to have those. I can can be done in separate pr.

Reply:
Good point. I think the `VotingPowerDisCache` in general needs comprehensive tests for which we have none. Created an issue for this #29 and will address it in a separate PR